### DISCUSSION TITLE
add folder structure by using '\' or '/'

### DISCUSSION THREAD

**Discussion by Aliaksandr3by** [doublecmd/doublecmd#1068] [Suggestions]
### Description

if I have the folder name as 'qwe123', I want to add a slash in folder name for to create another folder structures e.g.: 'qwe\123' ==> 'qwe' and subfolder '123'

![image](https://user-images.githubusercontent.com/11432724/208279465-04c132d5-bca5-40da-a6c6-834475d4a8d9.png)

as a result

![image](https://user-images.githubusercontent.com/11432724/208279515-2ee65a4f-d066-4e5b-8c34-9464333a1be3.png)

**Comment by Skif-off** [comment]
Version?
Win7x32/Xubuntu, DC 1.0.9/1.1.0: works fine ("\" in Windos, "/" in Linux).

P.S. Pardon my curiosity :) Why are you adding the label? It seemed to me that developers usually do it themselves.

**Comment by Aliaksandr3by** [comment]
windows 11_64
Double Commander
Version: 1.0.9
Revision: 222
Commit: a844456e4
Build date: 2022/12/03
Lazarus: 2.2.5.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 11 21H2 x86_64

![image](https://user-images.githubusercontent.com/11432724/208603574-134fdaf4-d37b-4c8c-bac5-7da99efe39f1.png)
![image](https://user-images.githubusercontent.com/11432724/208603612-a84e2678-f468-45de-b6bf-68547edd5705.png)

**Comment by Skif-off** [comment]
I am sorry :)) Today I think slowly, I thought about the F7 button...

**Comment by Aliaksandr3by** [comment]
In 'total commander' I used to this feature. I think it is very useful to rename a folder with add a '\\' symbol for creating a subfolder.

**Comment by avmaksimov** [comment]
I think it works fine now.

**Comment by msone** [comment]
No luck on macOS, neither in the file list nor the multi-rename tool. This is a key feature I miss from Total Commander. It would be a nice touch if Double Commander added it.
